README.md

```diff
@@ -74,18 +76,24 @@ spec:
     topic.name.badgeins: DOOR.BADGEIN
     topic.name.newcustomers: CUSTOMERS.NEW
     topic.name.sensorreadings: SENSOR.READINGS
+    topic.name.onlineorders: ONLINE.ORDERS.NEW
+    topic.name.outofstocks: OUT.OF.STOCK
+    
 
     #
     # format of timestamps to produce
     #
-    #    default is chosen to be suitable for use with Event Processing
+    #    default is chosen to be suitable for use with Event Processing,
     #    but you could modify this if you want to demo how to reformat
     #    timestamps to be compatible with Event Processing
     #
     #    NOTE: sensor readings topic is an exception. Events on this topic
     #           ignore this config option
     #
     formats.timestamps: yyyy-MM-dd HH:mm:ss.SSS
+    # format of timestamps with local time zone (UTC time in ISO 8601 format)
+    #    NOTE: this format is used by default for online orders 
+    formats.timestamps.ltz: yyyy-MM-dd'T'HH:mm:ss.SSS'Z'
```

Comment:
I'm a little skeptical about this... do we need to have the way to create events with a mixture of timezone formats?

Reply:
The product supports LTZ timestamps, but we did not have any tutorial using that format, so we though that it could be good to have some topics using that format and include them in the new tutorial 
The online orders topic is using this format